superset/tasks/scheduler.py

```diff
@@ -67,6 +68,8 @@ def execute(report_schedule_id: int, scheduled_dttm: str) -> None:
         AsyncExecuteReportScheduleCommand(report_schedule_id, scheduled_dttm_).run()
     except ReportScheduleUnexpectedError as ex:
         logger.error("An unexpected occurred while executing the report: %s", ex)
+    except SoftTimeLimitExceeded as ex:
```

Comment:
nit: this is probably unnecessary because we already catch and reraise it here: https://github.com/apache/superset/blob/master/superset/reports/commands/execute.py#L183